examples/pylint/W0716_wrong_exception_operation.py

```diff
@@ -0,0 +1,4 @@
+try:
+    1 / 0
+except ZeroDivisionError >= Exception:  # Error on this line
```

Comment:
for this one I would probably try `ZeroDivisionError or Exception`

Reply:
Hmm, looks like `or` causes a different exception to be changed. I will change this to `+`.